bindings/python/CMakeLists.txt

```diff
@@ -79,20 +75,18 @@ if ( NOT ${VALID_PIP_EXIT_CODE} EQUAL 0 )
     CODE
     "EXECUTE_PROCESS(
       COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install \
-                ${PIP_INSTALL_VERBOSE_FLAG} \
+                ${PIP_OPTIONS} \
                 --prefix \$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX} \
                 ${DEB_INSTALL_ARGS}
     )"
   )
 
 else()
-  # Use --force-reinstall to ensure a clean install if a rebuild needs to happen
   install(
     CODE
     "EXECUTE_PROCESS(
       COMMAND ${PYTHON_EXECUTABLE} -m pip install \
-                ${PIP_INSTALL_VERBOSE_FLAG} \
-                --force-reinstall \
+                ${PIP_OPTIONS} \
                 --prefix \$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX} \
```

Comment:
If we're taking `PIP_OPTIONS` to its logical extreme, we should also check to see if `--prefix` exists in `PIP_OPTIONS` and if so default to that instead of `\$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}` but we should also throw a warning to let the user know they're doing something probably bad!

Reply:
Can do this with https://cmake.org/cmake/help/latest/command/list.html#modification